scripts/base_tools.sh

```diff
@@ -77,14 +77,13 @@ as_root apt-get install -y --no-install-recommends \
         iputils-ping \
         jq \
         make \
-        python \
```

Comment:
The failure
```
+ repo init -u https://github.com/seL4/sel4test-manifest.git --depth=1
/usr/bin/env: 'python': No such file or directory
```
should be gone when we install the `python-dev-is-python3` package then also. See
https://packages.debian.org/bullseye/python-dev-is-python3

Reply:
fixed, thanks

Reply:
I wonder, maybe the packege `python-is-python3` is enough, this is what I usually use.

Reply:
you might be right, I thought it was a python-dev specific error but maybe not. I think the dockerifle should reflect the installation process of the docs, let me try python-is-python3.